### DISCUSSION TITLE
How to set proxies in Skyvern?

### DISCUSSION THREAD

**Discussion by timtim-hub** [Skyvern-AI/skyvern#1187] [Q&A]
I am using Skyvern and Docker, but I can't get to run a task with a proxy.

**Comment by suchintan** [comment]
I think we need to build some functionality to allow you to define your own proxy routes in the `.env` file that get loaded into the playwright browser initializations

We would love some OSS contirbutions here!

**Reply by KirschX** [reply]
Is there still no updates related yet with the custom proxy feature?

**Comment by martinarisk** [comment]
There is only residential proxy but did not found how to customize a proxy.
